bindings/python/test/environment/gravitational/earth/test_manager.py

```diff
@@ -1,14 +1,13 @@
 ################################################################################################################################################################
 
 # @project        Open Space Toolkit ▸ Physics
-# @file           bindings/python/test/environment/gravitational/earth/test_earth_manager.py
+# @file           bindings/python/test/environment/gravitational/earth/test_manager.py
```

Comment:
tbh I think there is a lot of naming problems in ostk cause all Earths have the same name, certain Celestials have the same name etc... don't we want to move in a direction where classes that do things for different other classes have different names? test_iers_manager or test_earth_manager is not much more difficult to write out and it improves clarity. Same thing with having an EarthGravitationalModel instead of just an Earth, because we end up referring to it as an EarthGravitationalModel in discussions on slack anyways.

Reply:
But the directory nesting + namespaces provides the necessary context, why bloating the file and class names?

Reply:
I see where you're coming from, but to me Earth vs EarthGravitationalModel doesn't sound like bloat and makes the whole library easier to navigate and program.